Workflow.xcworkspace/xcshareddata/xcschemes/Workflow.xcscheme

```diff
@@ -20,113 +20,43 @@
-               BuildableName = "libWorkflowLint.a"
-               BlueprintName = "WorkflowLint"
-               ReferencedContainer = "container:../WorkflowLint/WorkflowLint.xcodeproj">
-            </BuildableReference>
-         </BuildActionEntry>
-         <BuildActionEntry
-            buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
-            <BuildableReference
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "WorkflowUIKit"
-               BuildableName = "WorkflowUIKit"
-               BlueprintName = "WorkflowUIKit"
-               ReferencedContainer = "container:">
-            </BuildableReference>
-         </BuildActionEntry>
       </BuildActionEntries>
    </BuildAction>
    <TestAction
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
-      codeCoverageEnabled = "YES"
-      onlyGenerateCoverageForSpecifiedTargets = "YES">
-      <PreActions>
-         <ExecutionAction
-            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
-            <ActionContent
-               title = "Run Script"
-               scriptText = "# Type a script or drag a script file from your workspace to insert its path.&#10;swift package generate-xcodeproj --package-path=$SRCROOT&#10;"
-               shellToInvoke = "generate xcodeproj">
-               <EnvironmentBuildable>
-                  <BuildableReference
-                     BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "Workflow"
-                     BuildableName = "Workflow"
-                     BlueprintName = "Workflow"
-                     ReferencedContainer = "container:">
-                  </BuildableReference>
-               </EnvironmentBuildable>
-            </ActionContent>
-         </ExecutionAction>
-      </PreActions>
-      <CodeCoverageTargets>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "Workflow"
-            BuildableName = "Workflow"
-            BlueprintName = "Workflow"
-            ReferencedContainer = "container:">
-         </BuildableReference>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "WorkflowUIKit"
-            BuildableName = "WorkflowUIKit"
-            BlueprintName = "WorkflowUIKit"
-            ReferencedContainer = "container:">
-         </BuildableReference>
-      </CodeCoverageTargets>
-      <TestPlans>
-         <TestPlanReference
-            reference = "container:../Workflow.xctestplan"
-            default = "YES">
-         </TestPlanReference>
-      </TestPlans>
```

Comment:
Looks like you dropped the test plan.  This is one of the reasons why coverage generation step is failing.  Essentially you changed the structure of the DerivedData folder to now be a single level instead of nested in a Workflow* folder.

Additionally when we found this, it seems that only 94 of the 225 tests are running.

Reply:
@morganzellers and I are looking into this issue and what the fix looks like,  but it will need to be resolved before merging in.

Reply:
Test plan removal was a partially deliberate choice. Xcode 12.5 is getting much better about just opening Package.swift files and having it work. They're actually removing the xcodeproj generation, if you run it now you get a warning. 

Because of this it seems optimal to attempt to keep as close to what Xcode generates from the package.swift as is reasonably possible. That line of reasoning is what caused me to remove the test plan, it's not a hill I'll die on if we want to just bring it back.

Reply:
Also an important note, is that our pipeline is still running 12.4.  I'm not sure if that's MS lagging behind or if we have something specified.  Either way we'll have to keep that in consideration as we move forward.

Reply:
Yeah there's a big thread on Github about that. The official answer was something like "We are working on it, if you want better you can pay MacStadium or use your own runner". Apparently Azure DevOps has the same challenges right now. I suspect the new chipset and general VM issues with Big Sur are making it hard for everybody right now.

What you should be careful of is that you specify macos-latest which means as soon as they update your builds will start using 12.5. It *may* be worth specifying a version to avoid breakage, but that also means you have to come up with your own cadence to review their hosted machine updates and choose to update.

Reply:
https://github.com/actions/virtual-environments/issues/2486

Reply:
Ok since, you intentionally removed the TestPlan, and it doesn't seem to impact my Xcode which is on 12.4, we are fine with this change.